src/raven.js

```diff
@@ -870,4 +869,13 @@ function afterLoad() {
         Raven.config(RavenConfig.dsn, RavenConfig.config).install();
     }
 }
+
+function urlencode(o) {
+    var pairs = [];
+    each(o, function(key, value) {
+        pairs.push(encodeURIComponent(key)+'='+encodeURIComponent(value));
```

Comment:
Wanna configure eslint or whatever the new hotness is for here? Apparently this isn't covered by jshint. :)